apidoc/_sources/arcgis.features.managers.rst.txt

```diff
@@ -28,3 +28,24 @@ FeatureLayerManager
     :inherited-members:
     :members:
     :undoc-members:
+
+VersionManager
+--------------
+.. autoclass:: arcgis.features._version.VersionManager
+    :inherited-members:
+    :members:
+    :undoc-members:
+
+Version
+--------------
+.. autoclass:: arcgis.features._version.Version
+    :inherited-members:
+    :members:
+    :undoc-members:
+
+ParcelFabricManager

```

Comment:
@DavidJVitale We need to hide these classes in this release (VersionManager, Version, ParcelFabricManager)

Reply:
We only need to hide parcel fabric, not versioning.